pandas/core/window.py

```diff
@@ -1867,6 +1867,15 @@ def _constructor(self):
 
     def _get_window(self, other=None):
         obj = self._selected_obj
+        axis = self.obj._get_axis_number(self.axis)
+
+        # If axis == 0, leave `obj` alone.
+        # If axis == 1, transpose.
+        #
```

Comment:
rather than thins, can you just use ``_get_axis(self.axis)`` and then measure the length of it?

Reply:
Yep, done.